### ISSUE TITLE
Markdown inside HTML regression in 6.6.6

### ISSUE THREAD

**Issue by sapegin** [probablyup/markdown-to-jsx#185]
Markdown:

````
<details>
  <summary>Solution</summary>

  ```jsx
  import styled from 'styled-components';
  ```
</details>
````

6.6.5 renders is correctly, similar to GitHub:

<details>
  <summary>Solution</summary>

  ```jsx
  import styled from 'styled-components';
  ```
</details>

6.6.6 renders just text:

![image 2018-06-10 at 6 49 00 pm](https://user-images.githubusercontent.com/70067/41203974-f64c0e9c-6cde-11e8-8bd8-56dcb728836b.png)

**Comment by probablyup** [comment]
Thanks Artem. I’m on vacation for this week so I’d suggest just pinning to
the previous version until I have a chance to take a look.

**Comment by pnepe6** [comment]
Hello I encounter the same issue. Markdown inside HTML is not properly rendered.

Code in `.md` file:
```
<details>
<summary>View Changelog</summary>
<p>

# Title h1

Text content [link-to][link1].

[link1]: http://link1

    ```jsx
    import styled from 'styled-components';
    ```

</p>
</details>
```
It was working well on `react-styleguidist@6.5.0`.

But not anymore on `react-styleguidist@7.0.12`.

Is the usage changed or is it a regression?

Anyway, in all case many thanks for your awesome work, `react-styleguidist` rocks!

**Comment by pnepe6** [comment]
Hi,

I encounter a similar issue.

When using `markdown-to-jsx@6.6.5` (such as 6.0.2 until 6.6.4 and also 6.6.6)

I use the `compiler` provided to make a react component  `<Markdown />`:
```
import React from 'react';
import { compiler } from 'markdown-to-jsx';

const Markdown = (props) => (
    <div>
        {compiler(props.text, { forceBlock: true, parseBlockHtml: true })}
    </div>
);

export default Markdown;
```

And use it:
```
const text = `
<details>
<summary>View collapsed content</summary>

# Title h1

## Title h2

Text content

* list 1
* list 2
* list 3


</details>
`;

<Markdown text={text} />
```

Then it renders an incorrect markdown:
![mkd-not-work](https://user-images.githubusercontent.com/28387666/41532544-ab76e538-7321-11e8-85b1-3078e22d4116.png)

I expect to have the same render as in any markdown renderer such as in git:
![mkd-work](https://user-images.githubusercontent.com/28387666/41532570-beafe8ca-7321-11e8-8b13-1cf415ed3f0c.png)

I make a [repo example](https://github.com/pnepe6/test-react-styleguidist-markdown-not-work) you can check.

```
$ git clone git@github.com:pnepe6/test-react-styleguidist-markdown-not-work.git
$ cd test-react-styleguidist-markdown-not-work
$ npm install
$ npm run styleguide
```
Then check the result on your browser: `http://localhost:9090`

Is any update planned? Or maybe I don't use it correctly?

**Comment by probablyup** [comment]
I think that's the same bug @pnepe6. Looking into it today, hopefully will have an update soon.

**Comment by probablyup** [comment]
Fixed in `6.6.7`, thanks for your patience!
